.local/render-proto-sweep.py

```
@@ -0,0 +1,257 @@
+                             annotations. Matches render-sweep.py's chart language.
+        $DIR/summary.txt   — text table.
+
+If $DIR has no metrics.csv (older sweep format), falls back to the
+sweep.log + ch-growth.log retroactive parser.
+"""
+import sys, os, csv, glob, math, re
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+DIR = sys.argv[1] if len(sys.argv) > 1 else sorted(glob.glob('/tmp/proto-sweep-2*'))[-1]
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/proto-sweep-2*"

(S108)